pyramid_oereb/standard/pyramid_oereb.yml.mako

```diff
@@ -472,14 +472,15 @@ pyramid_oereb:
   # The extract is the entry point which binds everything
   # related to data together.
   extract:
-    # Information about the base data used for the extract, e.g. the used base map and its currentness.
+    # Last update of the cadastral survey used for the extract, e.g. the used base map and its currentness.
+    # Format: dd.mm.yyyy
     # This is a multilingual value. In the minimum, the value for the default language has to be defined.
     base_data:
         text:
-          de: "Daten der amtlichen Vermessung. Stand der amtlichen Vermessung: {0}."
-          fr: "Données de la mensuration officielle, état de la mensuration officielle: {0}."
-          it: "Dati della misurazione ufficiale. Stato della misurazione ufficiale: {0}."
-          rm: "Datas da la mesiraziun uffiziala. Versiun dal mesiraziun uffiziala: {0}."
+          de: "{0}"
```

Comment:
Now all the values are the same for `de`, `fr`, `it` and `rm`-> `"{0}"`.
Is a multilingual element still needed here?

Reply:
Thanks, I think you are right - im Rahmenmodell S. 37:

| Name  | Kardinalität | Typ | Beschreibung |
| ------------- | ------------- | ------------- | ------------- |
| StandAV | 1 | Datum | Aktualisierungsdatum der AV im Katasterportal |

I will try to change this

Reply:
{0} is just the update date of cadastral data. Sorry I didn't answer sooner.

Reply:
OK, that's what I understood - so would you agree that there's something to change in the data extract? In v1 we had an attribute `BaseData` which was a multilingual text, and in v2 the attribute is now `UpdateDateOS` of type datetime (not multilingual)

Reply:
It should already have been changed in the data extract otherwise we missed it 😁
Without checking in the specs, the data extract should (as you stated) just have the attribute `UpdateDateOS` of type _date_
No more text. - The label should come from the multilingual texte.xml or it's extension file mentionned on gitter (3.9.2021): https://files.gitter.im/60d9cb706da03739847fdc81/Uq6U/OeREBKRM_Vx_x_AuszugTexte_20210817.xml

Reply:
:+1: thanks. The label is being implemented on the MPF side (https://github.com/openoereb/pyramid_oereb_mfp/pull/79/files). I'll implemented the change `BaseData`-> `UpdateDateOS` in this PR (I checked also on master and we still have `BaseData`)